rbc/externals/__init__.py

```diff
@@ -48,4 +49,10 @@ def codegen(context, builder, sig, args):
     exec(fn_str, module_globals)
     fn = module_globals[fname]
     fn.__doc__ = doc
+
+    # https://chriswarrick.com/blog/2018/09/20/python-hackery-merging-signatures-of-two-python-functions/
+    # https://stackoverflow.com/questions/1409295/set-function-signature-in-python
```

Comment:
FYI, StackOverflow's license is not compatible with BSD. So it's better to not say anything like that.
```suggestion
```